compiler/next/include/chpl/queries/Context.h

```diff
@@ -615,6 +617,7 @@ class Context {
 
 
   /// \endcond
+  void gdbShouldBreakHere();
```

Comment:
Oh, because the debugger isn't necessarily gdb? That would be fine with me. It can call gdbShouldBreakHere :)

Reply:
I moved it to `util/break.h` and `util/break.cpp`. I created a `chpl::util` namespace for it because putting it in the plain `chpl` namespace collided with the existing `gdbShouldBreakHere()`. Maybe that wasn't the right thing to do, I can change it if we think it belongs somewhere else.
The existing `gdbShouldBreakHere()` is empty from what I can tell, so I _guess_ I copied the implementation?

Reply:
Right, I'm expecting you to leave it in the top level and adjust the production code to `#include "chpl/util/break.h"` it instead of defining gdbShouldBreakHere itself.

Reply:
Ok, I think I got it sorted out now. I removed the definition of `gdbShouldBreakHere` from the `misc.cpp` file in the production compiler, and then imported `break.h` there, and removed the namespaces from `break.h`.  I think this is what you intdended, but let me know if that's not what you're expecting.

Reply:
Looks good to me, thanks